elasticdl/python/common/model_utils.py

```diff
@@ -191,6 +191,31 @@ def get_model_spec(
     )
 
 
+def get_training_func_spec(
```

Comment:
`training_func_spec` is a new concept. Please add more description what it stands for.